backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/SyncToRemoteStatus.java

```diff
@@ -0,0 +1,10 @@
+package com.virtuslab.gitmachete.backend.api;
+
+public interface SyncToRemoteStatus {
```

Comment:
```suggestion
public interface ISyncToRemoteStatus {
```

Reply:
Btw maybe add a check for "I" prefix either to checkstyle (preferably) or via a script